arbitrum-docs/for-devs/troubleshooting-building.md

```diff
@@ -5,6 +5,8 @@ user_story: As a developer, I want to understand how to troubleshoot common issu
 content_type: troubleshooting
 ---
 
-import FAQStructuredData from '@site/src/components/FAQStructuredData';
+import FAQStructuredDataJsonLd from '@site/src/components/FAQStructuredData';
+import FAQQuestions from '../partials/_troubleshooting-building-partial.md';
 
-<FAQStructuredData faqsId="building" />
+<FAQStructuredDataJsonLd faqsId="building" />
+<FAQQuestions />
```

Comment:
Each FAQ page now imports and renders 2 elements:

The JSON-LD information for the structured data
The actual questions and answers